tests/aws/services/lambda_/event_source_mapping/test_lambda_integration_kinesis.py

```diff
@@ -535,9 +546,9 @@ def verify_failure_received():
 # TODO: add tests for different edge cases in filtering (e.g. message isn't json => needs to be dropped)
 # https://docs.aws.amazon.com/lambda/latest/dg/invocation-eventfiltering.html#filtering-kinesis
 class TestKinesisEventFiltering:
-    @pytest.mark.skipif(
-        is_v2_esm(), reason="DestinationConfig and LastProcessing result snapshot diff in ESM v2"
-    )
+    # @pytest.mark.skipif(
+    #     is_v2_esm(), reason="DestinationConfig and LastProcessing result snapshot diff in ESM v2"
+    # )
```

Comment:
Can this be deleted instead of commenting it out?

Reply:
ups, will look into it and either delete or keep it skipped 👀  (oversight in [62272ee](https://github.com/localstack/localstack/pull/11337/commits/62272eef1f3afe93e2119dd03c92f3d7579413e2))

Reply:
I'll leave it skipped for now because it's failing for another reason (little progress, but JSON filtering is still an issue as mentioned).